relational/table.py

```diff
@@ -144,6 +146,25 @@ def _get_dbi_units(proj):
     return None
 
 
+def _get_tco_units(proj):
+    """
+    Returns:
+      Net new units from TCO dataset (summation number of all unit numbers
+      from existing permits). None if no data in TCO.
+    """
+    num_tco_units = 0
+    tco_units_added = False
+    for root in proj.roots[TCO.NAME]:
```

Comment:
you might want to wait and/or merge with one of my branches, which adds a "fields" method to projects that fetches all entries

Reply:
Yup just saw this in your other PR, I'll wait on that change to go in

Reply:
Ok added